### ISSUE TITLE
It would be nice if you can provide a swatch of colors at the top (few colors to choose from) and the background becomes thats color

### ISSUE THREAD

**Issue by saintaze** [robots-ju/fll-timer#1]

**Comment by clarkwinkelmann** [comment]
If you are hosting the timer yourself, there is a variable in the source SASS file to easily change colors.

If you are referring to our hosted version, do you have a use case you could share ? From our analytics it seems the hosted scoreboard is only rarely used by external people, which wouldn't make the change be very worthwhile.

**Comment by saintaze** [comment]
yes, you might be correct. I just looked at the project and thought it was cool and was merely suggesting it as a bonus feature which might not really be needed.

**Comment by clarkwinkelmann** [comment]
I'll keep this open as a suggestion, but will probably only revisit it if we ever make other changes to the timer.

We have another internal version that's used in our scoring application that might be open-sourced at some point. It's unlikely for this vanilla-javascript scoreboard to be ever updated :sweat_smile:

**Comment by saintaze** [comment]
sounds good :P
